packages/fogbender/src/index.ts

```diff
@@ -73,8 +73,10 @@ export const createNewFogbender = (): Fogbender => {
     async isClientConfigured() {
       const snapshot = {
         getValue: () => state.events.configured,
-        subscribe: (cb: (s: Snapshot<boolean>) => void) => {
-          return state.events.on("configured", () => cb(snapshot));
+        subscribe: (cb: (value: boolean) => void) => {
+          const setState = () => cb(state.events.configured);
+          setState();
+          return state.events.on("configured", setState);
```

Comment:
Can't we replace this with
`cb(state.events.configured)
 return state.events.on('configure',cb)                          
`

Reply:
```suggestion
          cb(state.events.configured);
          return state.events.on("configured", cb);
```